library/ssl_tls13_keys.c

```diff
@@ -2,7 +2,7 @@
  *  TLS 1.3 key schedule
  *
  *  Copyright The Mbed TLS Contributors
- *  SPDX-License-Identifier: Apache-2.0
+ *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
  *
  *  Licensed under the Apache License, Version 2.0 ( the "License" ); you may
```

Comment:
The Apache blurb is still present here.

Reply:
This one is still present in f8be5f6adeb66b190ed063ecaddb7f73c339240a (both the `.c` and the `.h` file were concerned).